apps/user_status/src/components/SetStatusModal.vue

```diff
@@ -49,19 +49,21 @@
 					</button>
 				</EmojiPicker>
 				<CustomMessageInput
+					ref="customMessageInput"
 					:message="message"
-					@change="setMessage" />
+					@change="setMessage"
+					@submit="saveStatus" />
 			</div>
 			<PredefinedStatusesList
 				@selectStatus="selectPredefinedMessage" />
 			<ClearAtSelect
 				:clear-at="clearAt"
 				@selectClearAt="setClearAt" />
 			<div class="status-buttons">
-				<button class="status-buttons__select" @click="clearStatus">
+				<button class="status-buttons__select" :disabled="isSavingStatus" @click="clearStatus">
 					{{ $t('user_status', 'Clear status message') }}
 				</button>
-				<button class="status-buttons__primary primary" @click="saveStatus">
+				<button class="status-buttons__primary primary" :disabled="isSavingStatus || !message" @click="saveStatus">
```

Comment:
```suggestion
				<button class="status-buttons__primary primary" :disabled="isSavingStatus" @click="saveStatus">
```

Setting to empty clears the status just fine, might be less confusing

Reply:
It didn't in my instance and caused a server error as it was sending a status icon + empty message.
I personally wanted to set a status without message at that time, this is how I found the issue. Another user might think likewise. So if we fix it to clear the status on empty, the icon would not be set and would not match the user's expectation here.

Reply:
https://github.com/nextcloud/server/pull/23108

Reply:
ok, then it's either a regression or I found an action path that doesn't work:
<img width="862" alt="image" src="https://user-images.githubusercontent.com/277525/95841951-3f8cdc80-0d46-11eb-8a9e-d1b3a2ebe1c7.png">

I'll split this into a separate PR and keep this PR here for just the enter key improvements